nomos-blend/message/src/crypto/proofs/quota/inputs/prove/mod.rs

```diff
@@ -37,12 +37,8 @@ impl TryFrom<Inputs> for PoQWitnessInputs {
             index: value.private.key_index,
             leader_quota: value.public.leader_quota,
             message_key: (
-                fr_from_bytes(&signing_key_first_half[..]).expect(
-                    "First half of signing public key does not represent a valid `Fr` point.",
-                ),
-                fr_from_bytes(&signing_key_second_half[..]).expect(
-                    "Second half of signing public key does not represent a valid `Fr` point.",
-                ),
+                BigUint::from_bytes_le(&signing_key_first_half[..]).into(),
```

Comment:
Why this change? I think it's correct to use `fr_from_bytes`. If we don't care whether this is a valid `Fr` value, we can `expect`, but we should use the utility function exposed by the `groth16` crate.

Reply:
Nevermind, we are transforming a 16 byte array into an `Fr`, so this operation is guaranteed to succeed. We can use `.expect` in the result since we know it will always succeed.

Reply:
Got it. I saw that `fr_from_bytes` uses `BigUint::from_bytes_le` internally. Thanks. 
I reverted this change.